### PR TITLE
Fake data generation

### LINKED ISSUE #2: Write a script to add dummy data for testing
- write a script or other facility that will populate the database with a decent amount of random, sample data.

### PR DESCRIPTION
**Pull request by k0msenapati**
Closes #2 
Credits - https://free-api-app-web.vercel.app/

### CODE REVIEW

File: utils/seed.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        }
+
+        // Fetch all user IDs from the database
+        const allUsers = await client.user.findMany({
+            select: {
+                id: true,
+            },
+        })
+
+        const userIds = allUsers.map((user) => user.id)
```
Comment (kartikayasijaa) [inline utils/seed.js:55]: Remove it, It is not necessary.
you can simply use `const randomUserId = allUsers[Math.floor(Math.random() * allUsers.length)].id;`

File: utils/seed.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+        const colors = ["Violet", "Indigo", "Blue", "Green", "Yellow", "Orange", "Red"]
+        const sizes = ["XS", "S", "M", "L", "XL"]
+        const stores = [
```
Comment (kartikayasijaa) [inline utils/seed.js:59]: Please move all these constants in a seperate file, and import here

File: utils/seed.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+            "Wrap",
+            "Sheath",
+            "Fit and Flare",
+            "Maxi",
+            "Midi",
+            "Mini",
+        ]
+
+        // Insert products with random user IDs
+        for (const product of products) {
```
Comment (kartikayasijaa) [inline utils/seed.js:82]: Prepare product data array, and use createMany

**Review by kartikayasijaa — CHANGES_REQUESTED** [review]

File: utils/seed.js (hunk 3 above)
Comment (kom-senapati) [reply utils/seed.js]: These changes are done with commit dcbb5d6.

Comment (kom-senapati) [reply utils/seed.js]: Will try to convert files to .ts .

**Pull request closed without merging**